script/release

```diff
@@ -0,0 +1,83 @@
+#!/bin/bash
+
+#
+# Helper function to do replace; this should work across operating systems
+#
+
+function update {
+  TMP_FILE=`mktemp $1`
+  sed -e "$2" $3 > $TMP_FILE
+  chmod 0644 $TMP_FILE
+  mv -f $TMP_FILE $3
+}
+
+NEW_VERSION=$1
+REMOTE="release"
+REMOTE_URL="git@github.com:jrperritt/rack.git"
+VERSION_FILE="util/util.go"
+DOCS_INDEX_FILE="docs/index.rst"
+DOCS_CONFIGURATION_FILE="docs/configuration.rst"
+BRANCH=`git rev-parse --abbrev-ref HEAD`
+
+echo "Releasing new version: ${NEW_VERSION}"
+
+#
+# First confirm we're operating on the master branch
+#
+
+if [ "$BRANCH" != "master" ]; then
+  echo "Must release from master branch";
+  exit 1;
+fi
```

Comment:
Since a commit happens unconditionally below, we should also make sure that the workspace is clean:

```
diff=`git diff --exit-code`
if [ $? != 0 ]; then
  echo "..."
  exit 2
fi
```